lucene/build.gradle

```diff
@@ -15,8 +15,56 @@
  * limitations under the License.
  */
 
+// Should we do this as :lucene:packaging similar to how Solr does it?
+// Or is this fine here?
+
+plugins {
+  id 'distribution'
+}
+
 description = 'Parent project for Apache Lucene Core'
 
 subprojects {
   group "org.apache.lucene"
-}
\ No newline at end of file
+}
+
+distributions {
+  main {
+      // This is empirically wrong, but it is mostly a copy from `ant package-zip`
```

Comment:
This is wrong. It should rely on published artifacts from other subprojects (set up proper dependencies) and assemble them in the right way into the distribution layout.

Reply:
I tried looking at the solr packaging for figuring this out, but kept getting duplicate entries in the archive. I left the TODOs for what still remains to be added to the zip, can you give me a little more direction on what the implementation would look like?

Reply:
Sure but you'll have to be patient - can't do it right away.

Reply:
Haven't forgotten about it, just busy with work. Those release scripts will have to be adjusted to Solr and Lucene released independently in the future. Which requires independent builds, which requires repo split. Will have to get to it, eventually. Sigh.

Reply:
My goal here with getting things releasable is to also turn the smoke tester back on so that we can hopefully catch issues before we actually go to do the release. I understand there’s going to be more split related work, but that shouldn’t stop us from working on the pieces that we can work on before that.

Reply:
Ok, fair enough. But I wouldn't want to add things to gradle that are not right - these are hard to get rid of later on. I'm really busy this week but I will correct those assembly bits and commit them to this PR. Please give me some time to work on this, thank you.